### ISSUE TITLE
[Feature request] Map text

### ISSUE THREAD

**Issue by Duseylicious** [encounterplus/encounterplus#412]
I'd love to just be able to put text, or a text box, on the map.  A quick solution may be this - a Marker option that doesn't show any icon, but only has text.

**Comment by Taliesin1** [comment]
Yes, please — especially for the DM layer. I'd love to be able to cut and paste or just type in some stuff. Maybe on a semi-transparent box that can easily be moved around like any other element.

**Comment by Splinter714** [comment]
From previous similar issue #63
> @Rycochet commented on Mar 3, 2020
> Would be nice to be able to show text directly on the map without needing to open a popup for it - though having the popup automatically visible would also work.
> 
> Use case is for "README" type info on a map for starter adventures, without needing to make it part of the map image itself.
